src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs

```diff
@@ -168,7 +179,24 @@ public virtual Task InitializeAsync()
 
         public virtual async Task DisposeAsync()
         {
+#pragma warning disable CS0618
+            Dispose(true);
+#pragma warning restore CS0618
             await AfterAllAsync();
         }
+
+        /// <summary>
+        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
+        /// </summary>
+        /// <param name="disposing">
+        /// if set to <c>true</c> the method has been called directly or indirectly by a  user's code.
+        /// Managed and unmanaged resources will be disposed.<br /> if set to <c>false</c> the method
+        /// has been called by the runtime from inside the finalizer and only unmanaged resources can
+        ///  be disposed.
+        /// </param>
+        [Obsolete("Dispose(bool) is deprecated, please use DisposeAsync() instead")]
+        protected virtual void Dispose(bool disposing)
```

Comment:
lgtm

Reply:
Don't we still need to implement `IDisposable` on this class?

Reply:
Yes, we do https://github.com/akkadotnet/akka.net/blob/a45501991bd366a4aa7db60347760332b6cd30c4/src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs#L22

Reply:
Done